rclcpp_components/src/component_manager.cpp

```diff
@@ -152,17 +181,6 @@ ComponentManager::create_node_options(const std::shared_ptr<LoadNode::Request> r
     .parameter_overrides(parameters)
     .arguments(remap_rules);
 
-  for (const auto & a : request->extra_arguments) {
-    const rclcpp::Parameter extra_argument = rclcpp::Parameter::from_parameter_msg(a);
-    if (extra_argument.get_name() == "use_intra_process_comms") {
-      if (extra_argument.get_type() != rclcpp::ParameterType::PARAMETER_BOOL) {
-        throw ComponentManagerException(
-                "Extra component argument 'use_intra_process_comms' must be a boolean");
-      }
-      options.use_intra_process_comms(extra_argument.get_value<bool>());
-    }
-  }
-
```

Comment:
thanks for pointing this problem,  i agree with you that subclasses could be affected.  `use_intra_process_comms` and `use_dedicated_thread` both are also extra argument, so i wanted to avoid duplicated code by using map to search extra argument, but i didn't notice this problem.